engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.routes.js

```diff
@@ -26,4 +26,12 @@ angular.module('Bastion.capsule-content').config(['$stateProvider', '$urlRouterP
         }
     });
 
+    $urlRouterProvider.otherwise(function ($injector) {
```

Comment:
You're right, this is not a good approach because the `.otherwise()`is set globally.  I will try to find a different solution for this.